views/admin/after.jade

```diff
@@ -0,0 +1,17 @@
+extends ../layout
+
+block content
+
+  h3 Startzeit
+
+  form#payment-details(method='POST', action='after')
```

Comment:
yes. I fixed it in the functional tests that are not yet in a pr because they don't work...
will fix this. one sec